Tensile/Tests/extended/global_split_u/hgemm_gsu_minkforgsu.yaml

```diff
@@ -0,0 +1,65 @@
+GlobalParameters:
+  PrintLevel: 1
+  ForceRedoBenchmarkProblems: True
+  ForceRedoLibraryLogic: True
+  ForceRedoLibraryClient: True
+  CMakeBuildType: Release
+  EnqueuesPerSync: 1
+  SyncsPerBenchmark: 1
+  LibraryPrintDebug: False
+  NumElementsToValidate: 128
+  ValidationMaxToPrint: 16
+  ValidationPrintValids: False
+  ShortNames: False
+  MergeFiles: True
+  PlatformIdx: 0
```

Comment:
Please remove unrecognized global parameters.
PlatformIdx and DeviceIdx are unrecognized.